test/integ_test/evaluator/azureml_eval/conda.yaml

```diff
@@ -6,7 +6,7 @@ dependencies:
   - pip=22.3.1
   - pip:
       - azureml-defaults
-      - azureml-dataprep==4.8.4
+      - azureml-dataprep>=4.12.1
```

Comment:
remove this restriction?

Reply:
I cannot ensure there is no any other extra packages containing the restriction which may install the issued version, so I added that.